test/verify/check-docker

```diff
@@ -339,6 +339,16 @@ CMD ["/bin/sh"]
         b.wait_present("#container-terminal .console-ct .terminal")
 
         b.focus('#container-terminal .console-ct .terminal')
+
+        # Wait for the container to wake up
+        try:
+            b.key_press( [ 'Return', 'Return', 'Return' ] )
+            b.wait_in_text("#container-terminal", "#")
+        except Error, ex:
+            if not ex.msg.startswith('timeout'):
+                raise
```

Comment:
I don't know what's enough. My aim was to see if this fixes it. But I can remove the addition wait_in_text below.